irc/server.py

```diff
@@ -435,6 +435,25 @@ def handle_dump(self, params):
             for client in channel.clients:
                 print("     ", client.nick, client)
 
+    def handle_ison(self, params):
+        for nick in params.split(" "):
+            if len(params) == 0 or params.isspace():
```

Comment:
This check should happen before the nick for-loop because it has nothing to do with the content/intent of the loop. Or it should not be executed more than once.